Airports/EGGD/SMR/Geo.txt

```diff
@@ -1400,3 +1402,11 @@ N051.22.55.271 W002.43.56.852 N051.22.55.271 W002.43.56.852 smrWhite
 N051.22.55.271 W002.43.56.852 N051.22.55.448 W002.43.56.871 smrWhite
 N051.22.55.448 W002.43.56.871 N051.22.55.448 W002.43.56.871 smrWhite
 N051.22.55.448 W002.43.56.871 N051.22.55.465 W002.43.56.471 smrWhite
+
+; - stand_stop
+N051.23.08.018 W002.42.15.641 N051.23.08.014 W002.42.15.111 smrYellow
```

Comment:
This seems to duplicate an above line, so just need it once, see above comment also.